cypress/integration/service_page_spec.ts

```diff
@@ -0,0 +1,19 @@
+/// <reference types="cypress" />
+/* eslint-disable newline-per-chained-call */
+
+describe('Service Page', () => {
+  const serviceId = 1;
+
+  it('should load a basic page', async () => {
+    cy.request(`/api/services/${serviceId}`).should(r => {
+      expect(r.status).to.eq(200);
+      expect(r.body.service).to.include.all.keys('id', 'name', 'long_description');
+
+      cy.viewport(1200, 1800);
+      cy.visit(`/services/${serviceId}`)
+        .get('.listing--main header h1').first().should('have.text', r.body.service.name)
```

Comment:
I'm a little bit concerned about just making direct CSS selector queries in the tests, since they can get difficult to read and also difficult to maintain as the app changes. Also, what will this look like for pages that use CSS Modules, where the CSS class names get mangled?

I think extracting these selectors into Page Objects, like we were doing with the TestCafe tests, is important for making the tests more maintainable.

I'm not opposed to merging this as is for now, but I don't think straight CSS selectors will work very well with our Edit Page tests.